src/client/main.cpp

```diff
@@ -64,16 +86,44 @@ int main() {
 
   while (true) {
     std::string input;
+    ::std::cout << "Select from options:" << ::std::endl;
```

Comment:
Haha, nice little interactive option thing here. 

I feel like at this stage of our project, we will need 2 extra components:
1) Testing and Test Infrastructure
2) Configuration

If we have a simple test suite that can be run that will be nice.

Reply:
I would say focus on config first and then unit test infra.

Having unit tests for data store implementations would go a long way. Anyway, we should discuss these in our sync this week.